Surelog fails to build becuase cmake is confused about Python version

https://travis-ci.com/github/SymbiFlow/conda-packages/jobs/349642231#L3447

[Installs Python (gets 3.7.6)](https://travis-ci.com/github/SymbiFlow/conda-packages/jobs/349642231#L3582), later in cmake it has trouble finding it. In fact is claims it finds it, but then also not...

This might be an issue with conflicting Python versions on the build system ? Is there something that needs to be done to have cmake be more robust, or does the CI environment be changed ?

```
CMake Error at /tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/surelog_1592301850162/_build_env/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
3777  Could NOT find Python3 (missing: Python3_EXECUTABLE Interpreter) (found
3778  suitable version "3.7", minimum required is "3.3")
3779Call Stack (most recent call first):
3780  /tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/surelog_1592301850162/_build_env/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:445 (_FPHSA_FAILURE_MESSAGE)
3781  /tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/surelog_1592301850162/_build_env/share/cmake-3.17/Modules/FindPython/Support.cmake:2398 (find_package_handle_standard_args)
3782  /tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/surelog_1592301850162/_build_env/share/cmake-3.17/Modules/FindPython3.cmake:311 (include)
3783  CMakeLists.txt:45 (find_package)
```

--- a/surelog/build.sh
+++ b/surelog/build.sh
@@ -7,11 +7,9 @@ if [ x"$TRAVIS" = xtrue ]; then
   CPU_COUNT=2
 fi
 
-mkdir build && cd build
+export PKG_CONFIG_PATH="$BUILD_PREFIX/lib/pkgconfig/"
+export CXXFLAGS="$CXXFLAGS -I$BUILD_PREFIX/include"
+export LDFLAGS="$CXXFLAGS -L$BUILD_PREFIX/lib -lrt -ltinfo"
 
-# TODO(hzeller): even though gperftools are installed, -ltcmalloc is not found.
-LDFLAGS="-lrt" cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_VERBOSE_MAKEFILE=1
-
+make -j$CPU_COUNT
 make -j$CPU_COUNT install
-
-

--- a/surelog/meta.yaml
+++ b/surelog/meta.yaml
@@ -21,8 +21,6 @@ requirements:
     - {{ compiler('cxx') }}
     - cmake
     - pkg-config
-
-  host:
     - python {{ python }}
     - gperftools
     - tk

--- a/surelog/meta.yaml
+++ b/surelog/meta.yaml
@@ -19,18 +19,19 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - python {{ python }}
     - cmake
     - pkg-config
-    - python {{ python }}
-    - gperftools
-    - tk
-    - libunwind
     - libuuid
+    - gperftools
+    - flex
+    - libunwind
     - swig
 
   run:
-    - python
+    - python {{ python }}
     - gperftools
+    - libunwind
 
 about:
   home: https://github.com/alainmarcel/Surelog

